collector/docker/kindling-collector-config.yml

```diff
@@ -51,6 +51,12 @@ analyzers:
     # edge_events_window_size is the size of the duration window that seats the edge events.
     # The unit is second. The greater it is, the more data will be stored.
     edge_events_window_size: 2
+    #JavaTraceDeleteInterval is the interval for cleaning up expired data in javatraces.
+	  #The unit is seconds.
+    JavaTraceDeleteInterval: 10
+    #JavaTraceExpirationTime is the expiration time for data in javatraces.
+	  #The unit is seconds.
+		JavaTraceExpirationTime: 30
```

Comment:
We use the "snake_case" instead of the "CamelCase" in the configuration file. And take care of the indentation in a yaml file.